### ISSUE TITLE
we need lures

### ISSUE THREAD

**Issue by twizzy0** [SmashMC-Development/Suggestions#967]
I honestly and truly think lures should be unbanned, for example when I am HA hunting I do not want to be there for 2 days looking for a specific Pokémon, that gets boring and just played out, lures really aren’t a huge advantage, but they’re needed.

**Comment by morrow12** [comment]
I dont agree lures are extremely eco breaking. Also there not hard to get you can do a raid an get one so i rather it stay banned.

**Comment by manlybro2002** [comment]
I do not agree that it is eco breaking. All it will do is lower prices from maximum to minimum. It is weird to assume prices will  become trashed since we already got minimum prices on each server.

breeders will prob need to just make up the change in income by just breeding different types of BPs.

**Comment by jantjewieger** [comment]
Imo the lures should stay banned. It takes away the grind and can become a bit too overpowered (the ha lure for example)
